tools/nni_cmd/command_utils.py

```diff
@@ -3,10 +3,11 @@
 import os
 import signal
 import psutil
-from .common_utils import  print_error, print_normal, print_warning
+from .common_utils import print_error, print_normal, print_warning
+
 
 def check_output_command(file_path, head=None, tail=None):
-    '''call check_output command to read content from a file'''
+    """call check_output command to read content from a file"""
```

Comment:
since you changed docstring format, I recommend you update the docstring following the discussed format :)